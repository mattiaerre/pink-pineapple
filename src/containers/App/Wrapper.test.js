import React from 'react';
import { create } from 'react-test-renderer';
// import { matcher, serializer } from 'jest-styled-components';
import Wrapper from './Wrapper';

// expect.extend(matcher);
// expect.addSnapshotSerializer(serializer);

test('Wrapper', () => {
  const tree = create(<Wrapper />).toJSON();
  expect(tree).toMatchSnapshot();
});
