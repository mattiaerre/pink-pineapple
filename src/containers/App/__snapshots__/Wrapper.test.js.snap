// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wrapper 1`] = `


.fsWbGN {
  font-family: Consolas, monaco, monospace;
  padding: 2em;
}

.fsWbGN table {
  padding: 2em;
  border-collapse: collapse;
  border: 1px solid gray;
}

.fsWbGN tbody tr:nth-child(odd) {
  background: lightpink;
}

.fsWbGN tbody tr:nth-child(even) {
  background: papayawhip;
}

.fsWbGN th,
.fsWbGN td {
  padding: 1em;
  border: 0px;
}

<div
  className="sc-bdVaJa fsWbGN"
/>
`;
